packages/pr-antora-content-guidelines-checker/src/main.ts

```diff
@@ -18,7 +19,10 @@ async function run(): Promise<void> {
     const token = core.getInput("github-token");
     const octokit: InstanceType<typeof GitHub> = github.getOctokit(token);
     let actionResult: ActionResult[] = [];
-    const modifiedFiles: string[] = await getModifiedFiles(octokit);
+    const modifiedFiles: string[] = await getFilesFromPR(octokit, [
```

Comment:
**suggestion:** I suggest we add a comment to explain why we don't take the "deleted" files to make things clearer
```suggestion
    // The checks are done on the content of the files, so they must not be applied to deleted files whose content is no longer available
    const modifiedFiles: string[] = await getFilesFromPR(octokit, [
```